src/main/java/stream/flarebot/flarebot/util/ColorUtils.java

```diff
@@ -0,0 +1,11 @@
+package stream.flarebot.flarebot.util;
+
+import java.awt.Color;
+
+public class ColorUtils {
```

Comment:
UnDescriptive name, would name something like `ColourConstants` instead

Reply:
or FlareColor

Reply:
I mean for right now it only has constants, it likely won't in the future though.

Reply:
I would also like to point out the fact that we have `ColorEnum`, maybe the two should be merged in some way?